pkg/api/server/register_pods.go

```diff
@@ -286,9 +286,36 @@ func (s *APIServer) registerPodsHandlers(r *mux.Router) error {
 	//   200:
 	//     $ref: "#/responses/DocsPodTopResponse"
 	//   404:
-	//     $ref: "#/responses/NoSuchContainer"
+	//     $ref: "#/responses/NoSuchPod"
 	//   500:
 	//     $ref: "#/responses/InternalError"
 	r.Handle(VersionedPath("/libpod/pods/{name}/top"), s.APIHandler(libpod.PodTop)).Methods(http.MethodGet)
+	// swagger:operation GET /libpod/pods/stats pods statsPod
```

Comment:
```suggestion
	// swagger:operation GET /libpod/pods/stats libpod statsPod
```

Reply:
./hack/swagger-check wants us to use "pods":
```
hack/swagger-check

pkg/api/server/register_pods.go:
-       // swagger:operation GET /libpod/pods/stats libpod statsPod
+       // ................. ... .................. pods
        r.Handle(VersionedPath("/libpod/pods/stats"), s.APIHandler(libpod.PodStats)).Methods(http.MethodGet)
make: *** [Makefile:402: swagger-check] Error 1

```